tcms/testruns/templates/testruns/mutable.html

```diff
@@ -60,6 +60,14 @@
             </div>
         </div>
 
+        {% if errors %}
+            <div class="form-group">
+                <label class="col-lg-12" for="id_notes">{% trans "Errors" %}</label>
+                <div class="col-lg-12">
+                   {{ errors }}
+                </div>
+            </div>
+        {% endif %}
```

Comment:
If this is only to show errors from the build field (as shown in the test) then you need to add errors only to that particular field (also Product & Version). See how it is done for manager and default tester above.

Reply:
I used form.errors, so I think no matter which field has an error it will show up.

Reply:
This is true, but we don't show all form errors anywhere else. If you are seeing errors with a specific field then show the errors for that field in its own HTML group. Otherwise make sure the code handles all the input values where necessary.